### ISSUE TITLE
Creating NFT without collection on BSX

### ISSUE THREAD

**Issue by JustLuuuu** [kodadot/nft-gallery#5590]
Got this question from one artist if we are planning to add an option of minting NFT without then needing to mint collection first on Basilisk. 

Are we planning to have that?

**Comment by roiLeo** [comment]
> Are we planning to have that?

Not really, I don't think it's possible on BSX chain as it must be attached to a collection Id.

ref https://galacticcouncil.github.io/api-docs/Basilisk-Rococo/pallets/NFT/extrinsic/mint

**Comment by JustLuuuu** [comment]
> Not really, I don't think it's possible on BSX chain as it must be attached to a collection Id.
> 
> ref https://galacticcouncil.github.io/api-docs/Basilisk-Rococo/pallets/NFT/extrinsic/mint

Thank you for letting me know. I will tell the artist about that :)

**Comment by roiLeo** [comment]
Close as not planed
